website/pages/index.js

```diff
@@ -61,6 +62,16 @@ function Home() {
   const context = useContext(DocusaurusContext);
   const {siteConfig = {}} = context;
 
+  // TODO: (wrapper function) API so that user won't need to concatenate url manually
+  const feedbackUrl = `${siteConfig.baseUrl}feedback/`;
+  const gettingStartedUrl = `${siteConfig.baseUrl}docs/installation`;
+
+  useEffect(() => {
+    // Prefetch feedback pages & getting started pages
+    __docusaurus.prefetch(feedbackUrl);
+    __docusaurus.prefetch(gettingStartedUrl);
```

Comment:
As I've tested the latest `master`, it _sometimes_ is not the object it expects. The root node - the one with `id="__docusaurus"` - is a _named element_, therefore it's a global. If the code which sets `window.__docusaurus` runs _after_ this very line, an error is thrown here.

Reply:
Thanks for reporting @radekmie. It is very interesting to see someone checking out this PR. We will most likely rename since there is a conflict